### ISSUE TITLE
a11y: Download icon missing alt attribute

### ISSUE THREAD

**Issue by iamjolly** [18F/all_sorns#130]
## Description:
The download icon on the search results page is missing the `alt` attribute. 

## Screenshot:
![screenshot of download icon with missing alt](https://user-images.githubusercontent.com/1093423/105231126-9e22dd00-5b23-11eb-80b5-d1c978c3cabb.png)

## Location of code:
[/app/views/sorns/search.html.erb, Line 40](https://github.com/18F/all_sorns/blob/41c4787de09eff4ff2c745250df070efa14504e2/app/views/sorns/search.html.erb#L40)

## WCAG SC
[Success Criterion 1.1.1 Non-text Content (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html): All non-text content that is presented to the user has a text alternative that serves the equivalent purpose, except for the situations listed below.
...
### Decoration, Formatting, Invisible
If non-text content is pure decoration, is used only for visual formatting, or is not presented to users, then it is implemented in a way that it can be ignored by assistive technology.

## Recommended fix:
Since this icon is decorative and part of the link that reads "Download results as a CSV file", the `alt` attribute, when added, can be blank (or null) so that assistive technology will ignore the image. The code for this should look something like this:
```
<%= image_tag("Download_Icon.svg", alt: "")%>
```
